package.json

```diff
@@ -65,23 +65,23 @@
     "eslint": "^8.39.0",
     "eslint-plugin-simple-import-sort": "^8.0.0",
     "husky": "^8.0.1",
-    "jest": "^27.5.1",
+    "jest": "^29.7.0",
     "jest-expect-message": "^1.1.3",
-    "jest-extended": "^1.2.0",
-    "jest-mock": "^28.1.0",
+    "jest-extended": "^4.0.2",
+    "jest-mock": "^29.7.0",
     "nock": "^13.1.0",
     "pino-pretty": "^9.0.1",
     "prettier": "2.8.8",
     "tmp": "^0.2.1",
-    "ts-jest": "^27.0.7",
+    "ts-jest": "^29.1.1",
     "typescript": "^4.7.4"
   },
   "jest": {
     "coverageDirectory": "out/coverage",
-    "globals": {
-      "ts-jest": {
+    "transform": {
+      "\\.[jt]sx?$": [ "ts-jest", {
```

Comment:
using the default regex
https://jestjs.io/docs/configuration#transform-objectstring-pathtotransformer--pathtotransformer-object